performance/config/base-config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
wat doet dit?
over het algemeen denk ik best ergens een pagina met documentatie maken

Reply:
De configuratie voor sitespeed die gemeenschappelijk is. Zal ik een README.md onder de performance-folder steken? Confluence leest toch geen kat.

Reply:
README.md aangemaakt.